kolibri/core/serializers.py

```diff
@@ -7,6 +9,8 @@ class DateTimeTzField(DateTimeField):
     def to_internal_value(self, data):
         data = super(DateTimeTzField, self).to_internal_value(data)
         tz = timezone.get_current_timezone()
+        if not data.tzinfo:
+            data = timezone.make_aware(data, pytz.utc)
```

Comment:
This is missed by the code coverage, perhaps add a test of something that doesn't have tz info?